frontend/src/widgets/ReasonList.js

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Table } from '@trussworks/react-uswds';
+import moment from 'moment';
+import withWidgetData from './withWidgetData';
+import Container from '../components/Container';
+import './ReasonList.css';
+import 'uswds/dist/css/uswds.css';
+
+// import { DATE_FMT } from '../Constants';
+
+/*
+  Widgets only have to worry about presenting data. Filtering of the data happens at a
+  higher level, which is why this component is wrapped with `withWidgetData`. `withWidgetData`
+  takes care of fetching data, flagging the component as loading and handling errors while
+  fetching data. Widgets are only rendered after data has been successfully loaded from the
+  API. Note the `example` passed as a 2nd parameter to `withWidgetData` must match the widget
+  id in the backend `src/widgets/index.js` file or you will get 404s.
+*/
+
+function ReasonList({ data, dateRange }) {
+  const [dateHeaderRangeValue, setHeaderDateRangeValue] = useState('');
+
+  useEffect(() => {
```

Comment:
no changes required, just making a note here that we ended up having two implementations of this, fine to clean up post merge with one